django_material_demo/cms/polls/views/question.py

```diff
@@ -58,12 +58,12 @@ def clean(self):
 
 
 class ChoicesForm(FormSetForm):
-    layout = Layout(Row('choice_text', 'vote_count'))
+    layout = Layout(Row('choice_text_en', 'choice_text_zh_hant', 'vote_count'))
```

Comment:
Is there way we don't need to specify the locale suffix (i.e. `_en`, `_zh_hant`) every time? Imagine when you add or remove one locale, you will have to update every single places.

Reply:
Found a way to auto generate field names with the suffixes 👍